src/sections/GroupGettersInfoSection/GroupGettersInfoSection.tsx

```diff
@@ -0,0 +1,73 @@
+import { useState } from "react";
+import { useFormContext } from "react-hook-form";
+import type { FormValues } from "@/pages/OrderPage/OrderPage";
+import AddGetterModal from "@/components/AddGetterModal/AddGetterModal";
+import {
+  SectionWrapper,
+  SectionDivider,
+  SectionTitle,
+  AddGetterButton,
+  InputRow,
+  GetterList,
+  GetterListTable,
+  Text,
+} from "@/sections/GroupGettersInfoSection/GroupGettersInfoSection.style";
+
+const GroupGettersInfoSection = () => {
+  const { watch } = useFormContext<FormValues>();
+
+  const getters = watch("getters");
+  const [isModalOpen, setIsModalOpen] = useState(false);
+
+  return (
+    <>
+      <SectionDivider />
+      <SectionWrapper>
+        <InputRow>
+          <SectionTitle>받는 사람</SectionTitle>
+          <AddGetterButton type="button" onClick={() => setIsModalOpen(true)}>
+            {getters.length > 0 ? "수정" : "추가"}
+          </AddGetterButton>
+        </InputRow>
+        <AddGetterModal
+          open={isModalOpen}
+          onClose={() => setIsModalOpen(false)}
+          onConfirm={() => setIsModalOpen(false)}
+        />
```

Comment:
```suggestion
        {isModalOpen && <AddGetterModal
          open={isModalOpen}
          onClose={() => setIsModalOpen(false)}
          onConfirm={() => setIsModalOpen(false)}
        />}
```

와 같은 형태로 수정하면 AddGetterModal 내부 로직으로 들어가지 않고도 Modal의 open여부를 판단할 수 있어서 더 적절한 표현 방식일 것 같네요!